packages/xs-vat-worker/test/escapeCompartment.js

```diff
@@ -0,0 +1,32 @@
+/* global issueCommand, Compartment */
+
+const te = new TextEncoder();
```

Comment:
I’ve been pretty consistent about using the names `encoder` and `decoder` in this case elsewhere. In general, I think it’s worthwhile to avoid initialisms for readability, except in the most dense mathematical formulas.